esp.d.ts

```diff
@@ -54,6 +56,20 @@ export class SingleModelRouter<T> {
     observeEventsOn(object : any, methodPrefix?: String) : Disposable;
 }
 
+export interface EventContext {
+    currentStage:string;
+    isCanceled:boolean;

```

Comment:
@raybooysen these are actually getters, i.e. `get currentStage() { ... }`, is this TS syntax therefore correct?